datadog/resource_datadog_dashboard_dependencies_test.go

```diff
@@ -0,0 +1,166 @@
+package datadog
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"strings"
+	"testing"
+)
+
+func datadogDashboardDepConfig(uniq string) string {
+	return fmt.Sprintf(`
+resource "datadog_monitor" "foo" {
+  name = "Name for monitor foo"
```

Comment:
Please include `uniq` in the monitor name:
```suggestion
  name = "%s monitor"
```